airflow/www/views.py

```diff
@@ -3012,6 +3015,57 @@ def action_muldelete(self, items):
         self.update_redirect()
         return redirect(self.get_redirect())
 
+    @action(
+        'mulduplicate',
+        'Duplicate',
+        'Are you sure you want to duplicate the selected connections?',
+        single=False,
+    )
+    @provide_session
+    @auth.has_access(
+        [
+            (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_CONNECTION),
+        ]
+    )
+    def action_mulduplicate(self, connections, session=None):
+        """Duplicate Multiple connections"""
+        for selected_conn in connections:
+            new_conn_id = selected_conn.conn_id
+            matches = re.findall(r"_Copy\((\d+)\)", selected_conn.conn_id)
```

Comment:
Is it intended to find `_CopyX` *anywhere*? Since the logic below only always adds a suffix, I would expect this to only check for a suffix as well.

Reply:
This is a good point,
In case of something like **abc_copy1_xyz**,  we will currently end up doing **abc_copy2_xyz**, but **abc_copy1_xyz_copy1** should be the accepted behaviour.